harvard-university-of-leeds.csl

```diff
@@ -153,7 +157,7 @@
             <text term="in" text-case="capitalize-first" font-style="italic" suffix=":"/>
             <names variable="editor" delimiter=", " prefix=" " suffix=".">
               <name and="text" delimiter-precedes-last="never" initialize-with=". ">
-                <name-part name="family" text-case="uppercase"/>
+                <name-part name="family" text-case="capitalize-first"/>
```

Comment:
@adam3smith, isn't it also rather uncommon to change the capitalization of names to "capitalize-first"?